README.md

```diff
@@ -29,13 +33,37 @@ format that has all dependencies and build instructions for an exercise.
 
 ### Running Tests
 
-Rename the file `src/Example.hs` to match the module name and run:
+In order to be accepted by Travis-CI, every exercise must compile without
+warnings and the example solution must pass the tests without failures.
+
+To test an exercise, first you need to rename the file `src/Example.hs`
+to match the module name, but - because we already have a *stub solution*
+in place - we have to move it first, otherwise it will be overwritten.
+
+```bash
+mv src/ModuleName.hs stub
+mv src/Example.hs src/ModuleName.hs
+```
+
+Running `stack test --pedantic` compiles and run the tests with
+`-wAll -wError`, but unfortunately it doesn't recompile unchanged
```

Comment:
I think this is `-Wall` and `-Werror`. Unless this flag is case insensitive, in which case I learn something!